src/aac/templates/genplug/README.md.jinja2

```diff
@@ -0,0 +1,34 @@
+# {{plugin.name}}
+
+## Plugin Commands
+
+### Shell Command
+
+```bash
+(.env) $ aac {{plugin.name}}
+```
+
+### Command arguments
+
+- `-h, --help`: Shows a help message
+{%- for command in commands %}
+{%- if command.input is defined %}
+{%- for command_input in command.input %}
+- `{{command_input.name}}`: {% if command_input.description is defined %}{{command_input.description}}{% else %}TODO: add description{%- endif %}
+{%- endfor %}
+{%- endif %}
+{%- endfor %}
```

Comment:
It looks like the command scope isn't correct.

You should loop this entire section in your commands like:
```suggestion
{%- for command in commands %}
### Shell Command - {{command.name}}
{{command.description}}

Example shell usage:
\```bash  # Apparently I have to escape this or the suggestion breaks. 
(.env) $ aac {{command.name.name}} -h
\```

### Command arguments

- `-h, --help`: Shows a help message

{%- if command.input is defined %}
{%- for command_input in command.input %}
- `{{command_input.name}}`: {% if command_input.description is defined %}{{command_input.description}}{% else %}TODO: add description{%- endif %}
{%- endfor %}
{%- endif %}

{%- endfor %}
```

Reply:
Resolved with 1f38927